configuration/sitemaps.md

```diff
@@ -9,7 +9,7 @@ title: Sitemaps
 
 In openHAB a collection of [Things]({{base}}/concepts/things.html) and [Items]({{base}}/concepts/items.html) represent physical or logical objects in the user's home automation setup.
 Sitemaps are used to select and prepare these elements in order to compose a user-oriented presentation of this setup for various User Interfaces (UIs),
-including [BasicUI]({{base}}/addons/uis/basic/readme.html),
+including [BasicUI](/addons/ui/basic/),
```

Comment:
This one throws a 404

The complete url would be `https://www.openhab.org/docs/configuration/ui/basic/`

So it should be `{{base}}/configuration/ui/basic/`

Reply:
Of course, thanks for catching this